Assignment2/Saabertooth1-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,48 @@
+{
+  "@context": {
+      "@type": "person",
```

Comment:
You have to define it out of the context. Also, you might define a specific URI otherwise "person" will be added to the default service URL.